lms/djangoapps/grades/rest_api/v1/views.py

```diff
@@ -207,3 +217,307 @@ def _get_course(self, request, course_id):
     def get(self, request, course_id, *args, **kwargs):  # pylint: disable=arguments-differ
         course = self._get_course(request, course_id)
         return Response(GradingPolicySerializer(course.raw_grader, many=True).data)
+
+
+class CourseGradingStatus(GradeViewMixin, PaginatedAPIView):
+    """ Course Grading Status.
+    """
+    authentication_classes = (
+        JwtAuthentication,
+        BearerAuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser,
```

Comment:
@Agrendalath Removing these, makes the API inaccessible with a logged-in user. I was not able to make a get call on the browser because the platform was not able to authenticate the user.